lte/gateway/docker/services/python/Dockerfile

```diff
@@ -126,6 +127,7 @@ RUN apt-get update && apt-get install -y \
   python3-eventlet \
   python3-pystemd \
   python3-jinja2 \
+  python3-scapy \
```

Comment:
The magma artifactory is imho not setup here, i.e., this installs scapy from the ubuntu sources - is this intentionally?

Reply:
You're right, this should be moved down to where we have the magma artifactory. Then we can also use the right version (although that didn't seem to cause problems in integ tests).

Reply:
On the other hand - iiuc we aim for pydep not to build python3-scapy anymore, right? Then we only have it available from the ubuntu repos (good that magma still seems to work with this version).

Reply:
That's a good point, in my current solution for make it only works because the pydep version is already in the artifactory. Then maybe using the Ubuntu version and not pinning might be better... I'm unsure.

Reply:
This PR removes scapy from setup.py and from the lockfile, i.e., it will not be build anymore. I suggest to everywhere use the ubuntu version and see if something breaks.
The alternative would imho be to keep it in the lockfile with a huge comment why we still want to package scapy - sounds like technical debt that will break at some point.

Reply:
Let's not keep it in the lockfile, it would also leave it as a magma dependency.

Another alternative might be to packages scapy in build-magma.sh together with the new tool. Not sure how difficult that would be.

Reply:
Would be a one-liner - fpm has support for directly package a pip package (as used in pydep).

Reply:
We just discussed in person, it's probably fine to install the ubuntu version.